launch/soccer.launch.py

```diff
@@ -90,14 +118,24 @@ def generate_launch_description():
         on_exit=Shutdown(),
     )
 
-    ref_receiver = Node(
+    internal_ref_receiver = Node(
+        condition=IfCondition(PythonExpression([use_internal_ref])),
         package="rj_robocup",
         executable="internal_referee_node",
         output="screen",
         parameters=[config],
         on_exit=Shutdown(),
     )
 
+    external_ref_receiver = Node(
+        condition=IfCondition(PythonExpression(["not ", use_internal_ref])),
```

Comment:
trial and error led me to discover this weird syntax (no documentation for this stuff)

Reply:
we're just like ROS!